subprojects/model-core/src/integTest/groovy/org/gradle/api/provider/MapPropertyIntegrationTest.groovy

```diff
@@ -655,9 +658,13 @@ task thing {
         failure.assertHasCause("Cannot query the value of task ':thing' property 'prop' because it has no value available.")
     }
 
+    @Issue('https://github.com/gradle/gradle/issues/23014')
     def "can put flatmap of task output into map property"() {
+
+        assumeFalse("WIP", configFromExtension)
```

Comment:
❓ What does `WIP` imply here? Will there be a follow up PR to address this?

Asking because this is actually skipping test cases

Reply:
Yes, https://github.com/gradle/gradle/pull/25848

Reply:
Thanks!